agent/tool-scripts/unittests

```diff
@@ -134,13 +155,12 @@ if [[ ${?} -ne 0 ]]; then
     exit 1
 fi
 
-for tool in $(find -L ${_ts} -maxdepth 1 -perm 775); do
-    toolname="$(basename ${tool})"
-    if [[ "${toolname}" == "base-tool" ]]; then
-        continue
-    fi
-    create_mock "${toolname}"
-done
+# Mock out GOLANG setups
```

Comment:
just a question, why do we need to do a GOLAND setup for tool-scripts unittest?

Reply:
> just a question, why do we need to do a GOLANG setup for tool-scripts unittest?

We have tools that use `GOLANG`,`pprof`, for example.